.travis.yml

```diff
@@ -23,7 +23,11 @@ before_script:
   - phpenv config-rm xdebug.ini
 
 script:
-  - if [[ "$VALIDATION" == "true" ]]; then phpunit --testsuite validation; fi
-  - if [[ "$VALIDATION" != "true" ]]; then phpunit --testsuite invariants; fi
-  - if [[ "$VALIDATION" != "true" ]]; then phpunit --testsuite grammar; fi
-  - if [[ "$VALIDATION" != "true" ]]; then phpunit --testsuite api; fi
+  - |
+    if [[ $VALIDATION = true ]]; then
+      phpunit --testsuite validation
+    else
+      phpunit --testsuite invariants
+      phpunit --testsuite grammar
+      phpunit --testsuite api
+    fi
```

Comment:
FYI - it turns out that tests weren't failing properly after this change. See https://github.com/Microsoft/tolerant-php-parser/pull/100/commits/9f502a93a9385ee627a14047d9c973026205a0ea for the fix. 

https://www.gnu.org/software/bash/manual/bashref.html#The-Set-Builtin